### PR TITLE
Checkstyle

### PR DESCRIPTION
**Pull request by Tiihott**
Includes:
- Adds checkstyle maven plugin.
- Makes changes to the code according to the checkstyle enforced coding standards.

First commit adds checkstyle and following commits apply changes according to the enforced coding standards. Last commit has biggest individual changes.

Related issue #108.

### CODE REVIEW

**Comment by Tiihott** [comment]
Squashed all smaller commits to a single large commit.

File: src/main/java/com/teragrep/pth_06/planner/walker/conditions/ElementCondition.java [hunk 1]
```diff
@@ -92,6 +93,8 @@ public Condition condition() {
                 QueryCondition host = new HostCondition(value, operation, config.streamQuery());
                 condition = host.condition();
                 break;
+            default:
```
Comment (eemhu) [inline src/main/java/com/teragrep/pth_06/planner/walker/conditions/ElementCondition.java:96]: I think this function could use some logic cleanup; but that is for another issue.

**Review by eemhu — APPROVED** [review]
Looks good to me, build passes when cloned and run `mvn clean package`.

**Review by kortemik — APPROVED** [review]

**Comment by kortemik** [comment]
thank you for the refactoring!

**Pull request merged**